[2022-07-26 10:04:58,123] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_1 scheduled__2022-07-25T00:00:00+00:00 [queued]>
[2022-07-26 10:05:00,932] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_1 scheduled__2022-07-25T00:00:00+00:00 [queued]>
[2022-07-26 10:05:01,048] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 10:05:01,050] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-26 10:05:01,052] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 10:05:11,715] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): runme_1> on 2022-07-25 00:00:00+00:00
[2022-07-26 10:05:56,424] {standard_task_runner.py:52} INFO - Started process 1660 to run task
[2022-07-26 10:06:08,487] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_1', 'scheduled__2022-07-25T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpe31ib7p8', '--error-file', '/tmp/tmpcisq800b']
[2022-07-26 10:06:11,427] {standard_task_runner.py:80} INFO - Job 73: Subtask runme_1
[2022-07-26 10:06:23,012] {local_task_job.py:221} WARNING - State of this instance has been externally set to queued. Terminating instance.
[2022-07-26 10:06:23,510] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 1660. PIDs of all processes in the group: [1660]
[2022-07-26 10:06:23,521] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 1660
[2022-07-26 10:06:28,684] {process_utils.py:75} INFO - Process psutil.Process(pid=1660, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='10:05:53') (1660) terminated with exit code Negsignal.SIGTERM
